packages/editor/src/yaml-table.tsx

```diff
@@ -13,13 +13,24 @@ import './codicon.css';
 import './yaml-table.css';
 
 interface YAMLVariablesTableProps {
-    vscodeApi: WebviewApi<unknown>;
+    vscodeApi?: WebviewApi<unknown>;
```

Comment:
I assume this part needs to be part to from the vscode integration?

Reply:
Yes, this API is provided when the table is rendered in the vs code extension.